scripts/s3-support-bundles/s3_bundle_generate.sh

```diff
@@ -23,22 +23,25 @@
 # Generate Support bundle for S3Server #
 #######################################################
 
-USAGE="USAGE: bash $(basename "$0") <bundleid> <path>
+USAGE="USAGE: bash $(basename "$0") <bundleid> <path> <sgiamadmin_pass>
```

Comment:
is there any way we can fetch ldap password from utility instead of taking it as parameter.
e.g. like salt-utils calls or confstore calls.
Reason is user may run this support bundle generation script independently hence he might need to fetch ldap credentials before running this script.
Also CSM calls this script to collect support bundle hence they also need to fetch and provide this ldap credentials before calling support bundle script.

Reply:
We cannot use salt-utils for password generation, as now openldap password will be in confstore, provided by provisioner component. 

To read password from confstore config file, we need to know the path of the config file, which is not known.

@nileshgovande gave this suggestion that the callers of s3_bundle_generate.sh should provide the ldap password.

Reply:
done